src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -510,7 +510,7 @@ trait GenesisDataLoaderBuilder {
 
 trait SecureRandomBuilder {
   lazy val secureRandom: SecureRandom =
-    Config.secureRandomAlgo.map(SecureRandom.getInstance).getOrElse(new SecureRandom())
+    Config.secureRandomAlgo.flatMap(name => Try(SecureRandom.getInstance(name)).toOption).getOrElse(new SecureRandom())
```

Comment:
Doesn't this hide the fact that the selected algorithm is available and other one is instead being used?

Reply:
If I add logging about algorithm non available, will it be sufficient?

Reply:
Yes, given our current handling of config errors I think that with logging with error will be enough here